_posts/2025-05-13-developer-update.md

```diff
@@ -0,0 +1,78 @@
+  Ideally we would like to have GeoServer 3 available by year end.
+
+<img src="/img/posts/2.26/gs-milestones.svg" alt="GeoServer 3 Milestones" style="display:block; margin-left:auto; margin-right:auto; width:100%;"/>
+
+## CITE Certification
+
+A great deal of progress has been made on CITE Certification with the most recent [GeoServer 2.27.0 Release]({% post_url 2025-04-03-geoserver-2-27-0-released %}) passing tests! This is great for interoperability and project stability as the CITE tests act as external "blackbox" testing framework and verifies that GeoServer is operating as intended. 
+
+We are presently determining how to pay for certification:
+
+* OSGeo has negotiated a reduced rate of $150 annual cost per test.
+* We are prioritizing tests where we can act as a "reference implementation" (so no annual cost)
+
+While GeoServer presently "implements" these standards, our sponsorship level is not sufficient to allow us to feel comfortable paying an annual costs of between $450 and $900 for certification each year.
+
+**Is our community interested in sponsoring CITE certification?**
+
+We would like confirmation that certification is valuable to the community. If you think it is valuable please let us know, or if you are interesting in sponsoring certification please speak up!
+
+Many thanks to prior sponsors of [this activity]({% post_url 2019-11-14-cite-test-automation-request-for-proposal %}) including
+[Gaia3D](https://gaia3d.com), and [OSGeo:UK](https://uk.osgeo.org).
+
+## ImageN Online Sprint (May 26-27)
+
+The biggest GeoServer 3 Milestone 1 activity is restarting the ImageN project and combining forces with JAI-Ext for a new image processing engine:
+
+* ImageN represents the Oracle donation of the original Java Advanced Imaging codebase to the Eclipse Foundation (using a new name that does not contain "Java").
+* The [ImageN project](https://projects.eclipse.org/projects/technology.imagen) is being restarted, with Andrea and Daniele being recenlty added to the project.
+* Project [website](https://projects.eclipse.org/projects/technology.imagen) has been updated with a slightly revised scope to reflect addition of the JAI-Ext codebase
+* We will be cutting some unused functionality, such as RMI, and restructuring the the maven build to reflect some of the lessons learned with JAI-Ext and GeoTools build practicies.
+* Andrea has a rough project plan [here](https://github.com/aaime/imagen/wiki/JAI%E2%80%90Ext-GeoTools-GeoWebCache-GeoServer,-migration-to-ImageN) which we will capture as a [project board](https://github.com/orgs/eclipse-imagen/projects/1) in the weeks ahead.
+
+Communication over on [imagen-dev](https://accounts.eclipse.org/mailing-list/imagen-dev) mailing list, introduce yourself when signing up if you would like to take part.
+
+## Wicket 10 Prep
+
+A big accomplisment in the recent [GeoServer 2.27.0 Release]({% post_url 2025-04-03-geoserver-2-27-0-released %}) is progress made towards getting ready for Wicket 10.
```

Comment:
```suggestion
A big accomplishment in the recent [GeoServer 2.27.0 Release]({% post_url 2025-04-03-geoserver-2-27-0-released %}) is progress made towards getting ready for Wicket 10.
```

Reply:
Or maybe something like "is progress towards Wicket 10".

Reply:
Oh I know, I will use the GS3 project plan headings here so the blog post becomes an update on each goal.

Let me try that and see how it goes.